README.md

```diff
@@ -47,8 +47,14 @@ Karpenter provider for AKS can be used in two modes:
 * Only Linux nodes are supported.
 * Kubenet and Calico are not supported
 
+## Installation (NAP)
+
+Follow the instructions in the Node Auto Provisioning [documentation](https://learn.microsoft.com/en-gb/azure/aks/node-autoprovision?tabs=azure-cli).
+
 ## Installation (self-hosted)
 
+**⚠️ Warning ⚠️** We strongly recommend using [Node Auto Provisioning](#installation-nap) (aka managed Karpenter) instead of self-hosted.
```

Comment:
Should we add more about why? 

Bootstrap contract isn't tested by AKS. before publishing images to the self-hosted galleries
No support for v6 skus(much cheaper) 
No --dns-service-ip 
Manage your own node bootstrap token on upgrade which can break bootstrapping 
No upgrade mw support

Reply:
Not a bad idea but maybe up above where we talk about NAP mode?

Reply:
yeah that makes sense